kolibri/core/assets/src/state/modules/connection.js

```diff
@@ -2,8 +2,12 @@ export default {
   state: {
```

Comment:
I'd rather not have this live in here, mostly because the `connection` this vuex module refers to is the one between the browser and the backend, which is distinctly different from the one we are concerned with whether it is metered.

Maybe let's just move this out into a nice little composable all of its own?